package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "theme-liquid-docs",
```

Comment:
I believe we could follow the `@shopify/theme-liquid-docs`. Wdyt?

Reply:
Completely agree with all of these changes https://github.com/Shopify/theme-liquid-docs/pull/217/commits/9434c0f0e3b64658ec1a482ebfa006be6afcdfc1